riff

```diff
@@ -199,6 +200,26 @@ Options:
 EOF
 }
 
+function print_publish_usage() {
+cat <<EOF
+
+Publish data to a topic using the http-gateway.
+
+Usage:
+
+  riff publish -i <input-topic> -d <data> [--count <count>] [--pause <pause>] [-t]
+
+Options:
+
+  -i, --input: the name of the input topic (defaults to the name of the current directory)
+  -d, --data: the data to post to the http-gatewy using the input topic
+  --count: the number of times to post the data (defaults to 1)
+  --pause: the number of seconds to wait bewteen postings (defaults to 0)
+  -t, --tail: tail, waiting for the function response
```

Comment:
not crazy about "tail" since it's just waiting for a single result every time in this case, but I don't have a better suggestion at the moment

Reply:
I'm not crazy about it either, but we already had that option. Would `--follow` be better for both use cases?

Reply:
I guess we can stick with it for now. We'll likely make changes when we move to a binary CLI.